src/gt4py/next/iterator/tracing.py

```diff
@@ -19,6 +19,7 @@
 from gt4py.eve import Node
```

Comment:
Maybe can we extract yet another pr e.g. everything in iterator except in transforms. I would like to be able to focus on the core of the PR without all the noise around. Let's discuss it.